_plugins/build.rb

```diff
@@ -0,0 +1,8 @@
+`npm install`
+
+production = ENV['BRANCH'] == 'master' or ENV['BRANCH'] == 'staging' or ENV['BRANCH'] == 'js-optimization'
```

Comment:
Useless assignment to variable - `production`.<br>Line is too long. [106/80]<br>Use `||` instead of `or`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.